convert_case.py

```diff
@@ -1,13 +1,14 @@
-message = input("Please write the string you want to case-convert: ")
-option = input("Please choose from the following options: \n \
+message = str(input("Please write the string you want to case-convert: "))
```

Comment:
Hi @Pianomen ! Thank you for the PR! 🚀 Please review my suggestions.

```suggestion
message = input("Please write the string you want to case-convert: ")
```

Explanation:
I don't think there's a need for converting the input to a string because the input in the CLI already comes as a string, even if you provide a number.